translate/cloud-client/translate_v3_translate_text_with_model.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START translate_v3_translate_text_with_model]
+
+from google.cloud import translate
+
+
+def translate_text_with_model(
+    project_id="YOUR_PROJECT_ID",
+    model_id="YOUR_MODEL_ID",
+):
+    """Translates a given text using Translation custom model."""
+
+    client = translate.TranslationServiceClient()
+
+    parent = client.location_path(project_id, "us-central1")
+    model_path = "projects/{}/locations/{}/models/{}".format(
+        project_id, "us-central1", model_id
+    )
+
+    # Supported language codes: https://cloud.google.com/translate/docs/languages
+    text_to_translate = "That' il do it."
```

Comment:
shouldn't this be commented out or set to a variable that's passed into the function?

Reply:
@nnegrey and I went back on forth with this. since other translate text sample are inlining the input, we agreed to inline them.

Reply:
No, if you have to pass the above string in for testing, please don't inline. 
If you can just use the string "text_to_translate" then please inline.

Reply:
Okay I will pass the input as param, and wont inline then.